### ISSUE TITLE
DistilBERTRelationMatcher -  TypeError: __new__() missing 1 required positional argument: 'task'

### ISSUE THREAD

**Issue by gonzalezf** [epfl-nlp/kogito#1]
When attempting to use the DistilBERTRelationMatcher class from the kogito.core.processors.relation module, I encountered the following error:

```python
from kogito.core.processors.relation import DistilBERTRelationMatcher

DistilBERTRelationMatcher(name='distilbert_matcher')

```
Error: TypeError: __new__() missing 1 required positional argument: 'task'

Could you please provide advice on how to solve this issue? It seems that the DistilBERTRelationMatcher class does not support the task argument.

I encountered the same problem when attempting to use the BERTRelationMatcher and SWEMRelationMatcher classes as well.

**Comment by mismayil** [comment]
Hi, thanks for bringing this issue to our attention. I am looking into this now and will give you an update soon.

**Comment by gonzalezf** [comment]
Thank you!

**Comment by mismayil** [comment]
It turns out `torchmetrics` dependency was upgraded to a major version and it introduced a bug. I have released a new version 0.6.2 that should fix the error, let me know if it works now.

**Comment by gonzalezf** [comment]
Thank you! It is working now :)
